libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardDateRelated.spec.ts

```diff
@@ -0,0 +1,104 @@
+// (C) 2022 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { EditMode } from "../../tools/editMode";
+import { DashboardHeader } from "../../tools/dashboardHeader";
+import { DashboardMenu } from "../../tools/dashboardMenu";
+import { Widget } from "../../tools/widget";
+import { DateFilter } from "../../tools/dateFilter";
+import { KpiConfiguration } from "../../tools/kpiConfiguration";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
```

Comment:
We should move to index.ts in the support folder to use for all tests.

Reply:
not all tests need this